lib/alex_koin/slack_commands.ex

```diff
@@ -66,6 +67,13 @@ defmodule AlexKoin.SlackCommands do
     coin
   end
 
+  def remove_coins(from_wallet, amount) do
```

Comment:
This method has a lot of business logic that doesn't seem pertinent to Slack

Reply:
Yeah... this stuff is overdue for a larger re organization. I might try and shoe horn that into the next push for the new features I'd like to add.